opentreemap/treemap/js/src/imageUploadPanel.js

```diff
@@ -93,6 +101,20 @@ module.exports.init = function(options) {
         }
     });
 
+    fileupload.bind('fileuploadadd', function(e, data) {
+        data.process(function() {
+            var defer = $.Deferred();
+            _.each(data.files, function(file) {
+                if (file.size >= MAX_FILE_SIZE_BYTES) {
+                    toastr.error(file.name + ' exceeds the maximum file size of 1MB');
```

Comment:
We should pass this error text in through the init options so that we can translate it.

I would also like to keep the human readable version of the max file in a constant right below the numeric version, so they can easily be changed together if we decide to increase the file size limit.